torch/_inductor/runtime/static_cuda_launcher.py

```diff
@@ -2,13 +2,11 @@
 from typing import Any, Optional
 from typing_extensions import Unpack
 
-from torch.utils._ordered_set import OrderedSet
-
 from .triton_compat import ASTSource, CompiledKernel
 
 
 MAX_SHARED_MEMORY = 49152
-MAX_ARGS = 50
+MAX_ARGS = 120
```

Comment:
I'm confirming that increasing MAX_ARGS does not cause OOMs or affect kernel performance. 120 args * 8 bytes per args = 0.96 kb, which should be basically nothing.